cpp_exercises/day3/phoneBook/tests/tst_phonebook.cpp

```diff
@@ -12,12 +11,21 @@ class testPhoneBook : public QObject
     ~testPhoneBook();
 
 private slots:
+    void fill(PhoneBook& phoneBook);
```

Comment:
Attenzione: qua `fill()` ti diventa un test case se lo metti come `private slot`.
Mettilo come `private` oppure usa lo slot `init()`. In quest ultimo caso devi però mettere la phoneBook come membro della classe di test.